# Check that break inside function and outside loop throws a syntax error

# Output:
# #syntax_error#

# Exit:
# 100

# Program:

begin
  int f() is
    break ;
    return 1
  end

  int x = call f()
end
